Create lyft.json

johandekeulenaer opened the pull request.

johandekeulenaer committed:
Create lyft.json

Gdewilde commented on data/lyft.json:
```diff
@@ -0,0 +1,34 @@
+  "dataCenters": [
+    "US"
+  ],
+  "hostingPartners": [
```
we renamed this to hostingProviders

johandekeulenaer replied:
Has been updated in the edit template

Gdewilde committed:
Update lyft.json

Gdewilde commented on data/lyft.json:
```diff
@@ -0,0 +1,34 @@
+  "applicationUrl": "https://account.lyft.com/auth",    
+  "twitter": "lyft",
+  "countryHQ": "US",
+  "gdprReadyStatus": "nonCompliant",
```
Link to an article to support this claim

Gdewilde reviewed (commented):
Relevant link to add: https://www.theverge.com/2018/1/25/16934064/lyft-customer-data-abuse-allegations

Gdewilde committed:
Merge branch 'master' into johandekeulenaer-AddLyft

johandekeulenaer committed:
Update lyft.json

Currently their service is available in US and Canada but it's unclear how they treat data of EU Resisdents. Further reserch learns they have no specific reference to compliance or efforts to GDPR regulation. For now, we change their status to Unknown, might evolve in the future when more references come available.

Gdewilde committed:
Update lyft.json
